buildenvs/github-action.Dockerfile

```diff
@@ -3,17 +3,14 @@
 # Licensed under the BSD-3-Clause License (the "License").
 # You may not use this file except in compliance with the License.
 ARG GO_VERSION=1.20.6
-ARG DEBIAN_VERSION=bullseye-20221114
-ARG GCC_SUFFIX=
-ARG GCC_VERSION=12.2.0
+ARG DEBIAN_VERSION=bookworm-20230725
 ARG KRAFTKIT_VERSION=latest
 ARG QEMU_VERSION=7.2.4
 ARG REGISTRY=kraftkit.sh
-ARG UK_ARCH=x86_64
 
 FROM golang:${GO_VERSION}-bullseye AS build
 
-COPY tools/github-action /go/src/kraftkit.sh/tools/github-action
+COPY . /go/src/kraftkit.sh
```

Comment:
Should we copy the whole project instead of just the tool?

Reply:
This commit requires the whole source directory: https://github.com/unikraft/kraftkit/pull/692/commits/0315bb5ed2f38bf6040b79b45d5301021dee9768